### ISSUE TITLE
Label palette doesn't appear correctly on Linux

### ISSUE THREAD

**Issue by aecreations** [aecreations/clippings#50]
Clippings 5.0b2

The clipping label palette widget doesn't appear correctly on Linux; the individual menu items representing the label colours appear as blank -- selected or not.  This is confirmed to be happening on Ubuntu 15.04 (Unity graphical environment), and Fedora 23 (GNOME).  This is not occurring on Kubuntu 15.10 (KDE).

With Clippings 5.0 due at the end of November, the fastest solution would be to use an alternative label menu (with the colour chips as the menuitem icons) instead of the label palette widget, when Linux is detected as the OS.

As Linux ranks 2nd in OS platform usage (see [AMO stats for Clippings](https://addons.mozilla.org/en-US/firefox/addon/clippings/statistics/usage/os/?last=30)), this is a fairly severe bug.

**Comment by aecreations** [comment]
Since it's not realistic to be able to test on every possible platform that Firefox is available in, the label picker palette widget should only be used on Windows and Mac OS X, where it is confirmed to be working, and fall back on the alternative label picker menu on all other platforms.  In code, the following check should be performed:
```
let os = aeUtils.getOS();
if (os != "WINNT" && os != "Darwin") { ... }
```

**Comment by aecreations** [comment]
On Ubuntu 15.04 and Fedora 23, any icons defined for menuitems in a menu don't appear; the clipping and folder icons on the Clippings submenu are missing.  But this isn't happening on OpenSUSE, which is KDE based.  Maybe this is a limitation imposed by the desktop graphical environment?

**Comment by aecreations** [comment]
Fixed in Clippings 5.0rc2
